python/catkin/builder.py

```diff
@@ -286,10 +287,42 @@ def get_multiarch():
     return out.strip()
 
 
-def get_python_install_dir():
-    # this function returns the same value as the CMake variable PYTHON_INSTALL_DIR from catkin/cmake/python.cmake
-    relevant_subfolders = 2 if os.name == 'nt' else 3
-    return os.sep.join(get_python_lib().split(os.sep)[-relevant_subfolders:])
+def get_python_install_dir_tuple(cmake_args=None):
```

Comment:
Do you have a use case for the argument `cmake_args` in mind? Currently it is not being used anywhere therefore I would rather not implement it.